server/jetstream_api.go

```diff
@@ -834,6 +836,7 @@ func (a *Account) trackAPIErr() {
 		jsa.sendClusterUsageUpdate()
 		js := jsa.js
 		jsa.mu.Unlock()
+		atomic.AddInt64(&js.apiTotal, 1)
```

Comment:
Same for those two atomic ops. I don't think that they need to be done outside of jsa's lock.